### PR TITLE
Added base usage example for aiohttp server

### PR DESCRIPTION
**Pull request by rudyryk**
Simple example of usage with aiohttp server with GET and POST requests.

Btw, `Manager.update()` seems to be broken. I'd want to restore it first and then go with smooth deprecation process if (really) necessary. Hopefully I'll be able to prepare the next pull request soon.

Hi @kalombos :) Thank you for being such an active contributor and maintaner! Please take a look at this PR.

### CODE REVIEW

**Comment by kalombos** [comment]
Hi @rudyryk Glad to see you :) I totally agree there should be smooth deprecation process but are you sure ```Manager.update()``` is broken? Because the tests have passed and i did not change this part of code. 

In our projects we created  BaseModel classes similiar to [AioModel](https://peewee-async-lib.readthedocs.io/en/latest/peewee_async/api.html#peewee_async.AioModel) It's more convinient to use such class instead of the Manager. So i decided to create AioModel class with async versions of peewee.Model methods. These methods should have the same name like in peewee with prefix aio and provide the same functionality. When the AioModel class is ready, the Manager class will just repeat its functionality. All Manager methods will be proxy-methods like [this](https://github.com/05bit/peewee-async/blob/master/peewee_async.py#L429) one. That's why i decided to make it deprecated. The only unique functionality the Manager has is database swap. I don't know how to use it :) And i thought it is for very rare cases so i decided to remove it in the future. 

I will review your PR in a couple of days

**Comment by rudyryk** [comment]
> but are you sure Manager.update() is broken?

It's not completely broken but now it requires models to be subclassed from `AioMode`. Actually the original intention of having the `Manager` was the ability to work without changing models hierarchy.

But I guess no big problem here, I've almost prepared a new PR with compatibility module extracted + tests.

**Comment by rudyryk** [comment]
`Manager` extracted to compatibility module in PR #232

**Comment by rudyryk** [comment]
This is first PR in a chain:
- #231 
- #232
- #233

File: examples/aiohttp_example.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+logger = logging.getLogger(__name__)
+
+database = PooledPostgresqlDatabase(
+    'examples', user='postgres', password='postgres',
+    host='db-postgres', port=5432,
+    min_connections=10, max_connections=100,
+)
+
+objects = Manager(database)
```
Comment (kalombos) [inline examples/aiohttp_example.py:19]: Are you sure you want to use Manager and api we are going to remove it future version?

File: examples/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  aiohttp:
+    build:
+      context: ..
+      dockerfile: examples/Dockerfile
+    depends_on:
+      - db-postgres
+    ports:
+      - 127.0.0.1:10080:10080
+    volumes:
+      - ../examples:/app/examples
```
Comment (kalombos) [inline examples/docker-compose.yaml:21]: Why not just put peewee_async in requirements? You want to use the example in the development proccess of peewee-async?

File: README.md [hunk 3]
```diff
@@ -99,35 +100,54 @@ with objects.allow_sync():
...
 ```
 
+
+More examples
+-------------
+
+Build and run with Docker Compose:
+
+```bash
+docker compose -f examples/docker-compose.yaml build
```
Comment (kalombos) [inline README.md:110]: Do we really need one more docker config for the example? Can we use the development one?

File: examples/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+fastapi==0.111.0
+uvicorn==0.29.0
+tornado==6.4
+aiohttp==3.9.5
+uvicorn==0.29.0
```
Comment (kalombos) [inline examples/requirements.txt:5]: uvicorn twice

File: examples/aiohttp_example.py [hunk 5]
```diff
@@ -0,0 +1,104 @@
+import json
+import logging
+import os
+# from asyncio import sleep as async_sleep
```
Comment (kalombos) [inline examples/aiohttp_example.py:4]: should be removed

File: examples/docker-compose.yaml (hunk 2 above)
Comment (kalombos) [reply examples/docker-compose.yaml:21]: Found your comment this is for live-reload while developing. I would not use docker for the application only for database The docker config for database already exists. The application can be run by bash command

File: examples/aiohttp_example.py (hunk 1 above)
Comment (rudyryk) [reply examples/aiohttp_example.py:19]: @kalombos I think we should have example for that and keep it explicitly unitl some version. I know projects that are using peewee-async and we should not break it. But I agree we'd need to communicate that properly.

Let's leave this example as-is and for future examples and documentation we'll mention that this API will be frozen in some version like `0.99` and removed in `1.0` or something.

Comment (kalombos) [reply examples/aiohttp_example.py:19]: Oh, i got your idea. Let's leave the example no problem :)

**Pull request merged**